spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe "Customers Show Page", type: :feature do
+    
+    # User Story 1    
+    it "displays a customer's name" do
+        king_soopers = Supermarket.create!(name: "King Soopers", location: "Denver")
+
+        joe = king_soopers.customers.create!(name: "Joe")
+
+        milk = king_soopers.items.create!(name: "milk", price: 5)
+        bread = king_soopers.items.create!(name: "bread", price: 3)
+        cheese = king_soopers.items.create!(name: "cheese", price: 4)
+
+        joe.items << milk
+        joe.items << bread
+
+        visit "/customers/#{joe.id}"
+
+        expect(page).to have_content("Joe")
+        expect(page).to have_content("milk")
+        expect(page).to have_content ("$3") #bread price
+        expect(page).to have_content("King Soopers")
+    end
+
+    # User Story 1
+    it "displays a customer's item and its information" do
+        king_soopers = Supermarket.create!(name: "King Soopers", location: "Denver")
+
+        joe = king_soopers.customers.create!(name: "Joe")
+
+        milk = king_soopers.items.create!(name: "milk", price: 5)
+        bread = king_soopers.items.create!(name: "bread", price: 3)
+        cheese = king_soopers.items.create!(name: "cheese", price: 4)
+
+        joe.items << milk
+        joe.items << bread
+        joe.items << cheese
+
+        visit "/customers/#{joe.id}"
+
+        expect(page).to have_content("milk")
+        expect(page).to have_content("$5")
+        expect(page).to have_content("King Soopers")
```

Comment:
I think you could consolidate these first 2 tests and test for the presence of all the items that Joe has -- milk, bread and cheese. The first test is just confirming that Joe's name is displayed, but you could test that in the same test as the presence of item information IMO